src/v2/Apps/Artwork/__tests__/ArtworkApp.jest.tsx

```diff
@@ -81,4 +81,44 @@ describe("ArtworkApp", () => {
       expect(wrapper.find("CascadingEndTimesBanner").exists()).toBeFalsy()
     })
   })
+
+  describe("explanatory banner for extended end times", () => {
+    it("includes banner when extended end times are enabled", () => {
+      const wrapper = getWrapper({
+        Sale: () => ({
+          cascadingEndTimeIntervalMinutes: 1,
+          extendedBiddingIntervalMinutes: 2,
+        }),
+      })
+      expect(wrapper.text()).toContain(
+        "Closing times may be extended due to last minute competitive bidding"
+      )
+    })
+
+    it("hides banner when extended and cascading bidding are disabled", () => {
+      const wrapper = getWrapper({
+        Sale: () => ({
+          cascadingEndTimeIntervalMinutes: null,
+          extendedBiddingIntervalMinutes: null,
+        }),
+      })
+      expect(wrapper.find("CascadingEndTimesBanner").exists()).toBeFalsy()
+      expect(wrapper.text()).not.toContain(
```

Comment:
#minor The above (the component not being rendered), is prob sufficient to assert (no need for the text part which feels somewhat brittle/not adding a lot)